vnext/src/RNTester/AlertExampleUwp.tsx

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ * @format
+ */
+
+import React = require('react');
+import {Text, View, Button} from 'react-native';
+import {Alert} from '../index.uwp';
+
+class AlertExample extends React.Component {
+  state = {
+    buttonPressed: 'none',
+  };
+
+  _threeButtons = () => {
+    var _this = this;
+
+    Alert.alert(
+      'Alert Title',
+      'Place the alert message here',
+      [
+        {
+          text: 'Ask me later',
+          onPress: () => _this.setState({buttonPressed: 'Ask me later'}),
+        },
+        {
+          text: 'Cancel',
+          onPress: () => _this.setState({buttonPressed: 'Cancel'}),
+          style: 'cancel',
+        },
+        {
+          text: 'OK',
+          onPress: () => _this.setState({buttonPressed: 'OK'}),
+        },
+      ],
+      {cancelable: false},
+    );
+  };
+
+  _twoButtons = () => {
+    var _this = this;
+
+    Alert.alert(
+      'Alert Title',
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
+      [
+        {
+          text: 'Cancel',
+          onPress: () => _this.setState({buttonPressed: 'Cancel'}),
+          style: 'cancel',
+        },
+        {
+          text: 'OK',
+          onPress: () => _this.setState({buttonPressed: 'OK'}),
+        },
+      ],
+      {cancelable: false},
+    );
+  };
+
+  _oneButton = () => {
+    var _this = this;
+
+    Alert.alert(
+      'Alert Title',
+      'Place the alert message here',
+      [
+        {
+          text: 'OK',
+          onPress: () => _this.setState({buttonPressed: 'OK'}),
+        },
+      ],
+      {cancelable: false},
+    );
+  };
+
+  render() {
+    return (
+      <View style={{flex: 1}}>
+        <Text style={{fontSize: 14}}>Click Button to call alert</Text>
+        <View style={{marginTop: 10}} />
+        <Button onPress={this._oneButton} title={'Alert with one button'} />
+        <View style={{marginTop: 10}} />
+        <Button
+          onPress={this._twoButtons}
+          title={'Alert with two buttons and larger message'}
+        />
+        <View style={{marginTop: 10}} />
+        <Button
+          onPress={this._threeButtons}
+          title={'Alert with three buttons'}
+        />
+        <Text style={{fontSize: 14, marginTop: 10}}>
```

Comment:
>{{fontSize: 14, marginTop: 10}} [](start = 20, length = 31)

You could consider pulling this style out to clean this up a little. It's not a lot of code though so probably not a big deal either way.